### DISCUSSION TITLE
Can we make unique validation?

### DISCUSSION THREAD

**Discussion by manyang-megantoro** [glhd/aire#109] [General]
as I know, aire use validatorjs, and in validatorjs have "registerAsync", so how I can use unique validation laravel with aire which is I know it use async validation

**Comment by inxilpro** [comment]
Generally, I just leave any server-side validations to just run on the server upon submit. That said, Aire just puts the `validatorjs` object at `window.Validator` — so you can add custom async validations to it manually, if you would like. Something like:

```js
window.Validator.registerAsync('unique', (value, attribute, requirement, passes) => {
  // Send value to server via AJAX
  // Call passes() with outcome
});
```
